### DISCUSSION TITLE
View Printer Driver Name and Port Name Without Remote PowerShell

### DISCUSSION THREAD

**Discussion by aljaz-h** [amidaware/tacticalrmm#2045] [Ideas]
I’m wondering if it would be possible to add a feature that allows viewing the printer driver name and port name programmatically, without needing to go into remote PowerShell sessions every time. This would be extremely helpful for system administrators and developers who manage multiple printers and often need to troubleshoot or gather configuration details.

Being able to retrieve this information easily through an API or tool would save a lot of time and simplify printer management, especially in larger environments.

Has anyone worked on or considered a solution like this? I'd love to hear any thoughts or suggestions on how feasible it might be.

Thanks in advance!

**Comment by dinger1986** [comment]
Run the community script to get printer info and save that to a custom field
